### PR TITLE
fix: tags support for included tasks

### LINKED ISSUE #86: Tags are not working
I wanted to quickly update my Prometheus config using the tag "prometheus_config". The playbook didn't work because the role uses "include_tasks".

I see 2 solutions:
1) Convert `include_tasks` to `import_tasks`.
2) Add `apply: {tags: ...}` to all ìnclude_tasks` call.

Is there a reason why this collection prefer `include_tasks`?

### PR DESCRIPTION
**Pull request by laurent-indermuehle**
This is a PR to fix #86

I think `import_tasks` should be prefered. But I don't want to impose my choice.

### CODE REVIEW

**Comment by gardar** [comment]
Thanks for the PR.
You are correct that currently the tags do very little, since they only apply to the include itself and the tags are missing from the included tasks.
The roles previously used the now deprecated `include` module and I had not tested the tags after migrating to `include_tasks`.

Rather than applying the tags to everything I would prefer that the tags were added to the relevant tasks instead, as applying the tags to the whole import/include will run some tasks that don't necessarily need to run.

File: changelogs/fragments/fix86_apply_tags.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - all roles - Fix tags to enable the execution of a specific tasks file as intended.
```
Comment (gardar) [inline changelogs/fragments/fix86_apply_tags.yml:3]: This is not needed as the changelog is automatically generated from the PR title, please update the title with the relevant tag.
See here: https://github.com/prometheus-community/ansible/blob/main/CONTRIBUTING.md#releases

**Review by gardar — CHANGES_REQUESTED** [review]

**Comment by laurent-indermuehle** [comment]
I discovered, thanks to the bot, that my commits must be signed-off. I just did. 

I think the contributors guide is lacking this information. But I appreciate the effort you make to automate quality control.

Ready for review.

**Comment by laurent-indermuehle** [comment]
I fixed a yaml indentation (thanks to ansible-lint, I love this tool). But I got trapped again by the signed-off of my commit. This is not documented in the contributors guide either.
Could you please remove this constrain or explain how to configure this repository to always signed-off commits?

Also, I did what your requested in your code review and resolved the comment, but this PR is still marked "requested changes". Do you know why?

**Review by gardar — APPROVED** [review]

**Pull request merged**